src/cai_lab_to_nwb/zaki_2024/interfaces/zaki_2024_edf_interface.py

```diff
@@ -1,33 +1,29 @@
-from pydantic import FilePath
 from pathlib import Path
-
 from neuroconv.basedatainterface import BaseDataInterface
 from pynwb import NWBFile, TimeSeries
 from pynwb.device import Device
-
 from mne.io import read_raw_edf
+from datetime import datetime, timedelta
+import numpy as np
 
 
 class Zaki2024EDFInterface(BaseDataInterface):
-
-    def __init__(self, file_path: FilePath, verbose: bool = False):
-
+    def __init__(
+        self,
+        file_path: Path,
+        start_datetime_timestamp: datetime = None,
+        stop_datetime_timestamp: datetime = None,
+        verbose: bool = False,
+    ):
```

Comment:
Also, if you add them here or in the conversion options (add_to_nwbfile) I think a docstring would be highly beneficial.